jina/clients/base/http.py

```diff
@@ -104,14 +104,14 @@ async def _get_results(
                 HTTPClientlet(url=url, logger=self.logger, **kwargs)
             )
 
-            def _request_handler(request: 'Request') -> 'asyncio.Future':
+            def _request_handler(request: 'Request') -> 'Tuple[asyncio.Future, None]':
```

Comment:
```suggestion
            def _request_handler(request: 'Request') ->Tuple[asyncio.Future,  Optional[asyncio.Future]]':
```

I think that it is better to stay consistent with the _request_handler from grpc and for claritiy to say that the type of the second part of tuple is an optional future even if we know it will be None.